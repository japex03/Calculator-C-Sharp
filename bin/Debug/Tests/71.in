sin(3.14/4)-sqrt(2)/2