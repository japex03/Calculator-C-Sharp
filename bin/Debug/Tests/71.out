-0,0002816